java/opendcs/src/main/java/org/opendcs/database/JdbiTransaction.java

```diff
@@ -0,0 +1,63 @@
+package org.opendcs.database;
+
+import java.sql.Connection;
+import java.util.Optional;
+
+import org.jdbi.v3.core.Handle;
+import org.opendcs.database.api.DataTransaction;
+import org.opendcs.database.api.OpenDcsDataException;
+
+import opendcs.util.sql.WrappedConnection;
+
+public class JdbiTransaction implements DataTransaction
```

Comment:
Should this be autoclosable?  Who is in charge of closing the handle?

Reply:
yes, the `DataTransaction` is in charge of ending the transaction. with the way the Handle instance is retrieved some portion of the calling entity is in charge.